cdap-docs/admin-manual/source/operations/preferences.rst

```diff
@@ -76,4 +76,32 @@ or provide Twitter API credentials::
       queue = new LinkedBlockingQueue<Tweet>(10000);
       collector = new CollectingThread();
       collector.start();
-    }
\ No newline at end of file
+    }
+
+Scoped Runtime Arguments for Workflow
+=====================================
+When Workflow is configured, you may want to pass different runtime arguments to different programs
```

Comment:
When a Workflow is configured, you may want to pass different runtime arguments to the different programs